lib/src/widgets/html_editor_widget_web.dart

```diff
@@ -818,6 +825,30 @@ class _HtmlEditorWidgetWebState extends State<HtmlEditorWidget> {
           });\n
         """;
     }
+    if (c.onEditLink != null) {
+      callbacks =
+      """$callbacks          \$('#summernote-2').on('summernote.mouseup', function(_, context) {
+            \$('.note-link-popover').off('click').on('click', function(e) {
+              if (e.target.className == 'note-icon-link' || (e.target.className == 'note-btn' && e.target.getAttribute('aria-label') == 'Edit')) {
```

Comment:
I'm worried if the editor supports multiple languages. Is the value `e.target.getAttribute('aria-label')` always equal to `'Edit'`?

Reply:
I found better way is to check the `targetChildren` tag if `targetClass` is `note-btn`. It took care of the issue.